.zenodo.json

```diff
@@ -87,6 +105,21 @@
         {
             "name": "Roy Smart"
         },
+        {
+            "name": "Mihail Bankov"
+        },
+        {
+            "name": "P. L. Lim",
+            "affiliation": "Space Telescope Science Institute, USA"
```

Comment:
@pllim you have an orcid you want to include here?

Reply:
```suggestion
            "orcid": "0000-0003-0079-4114",
            "affiliation": "Space Telescope Science Institute, USA"
```

Reply:
No problem!

Reply:
p.s. I am going to unsubscribe now but feel free to ping me again if you need more stuff from me. Good luck with the release!